src/sql/src/plan/statement/ddl.rs

```diff
@@ -2350,106 +2343,6 @@ pub fn plan_create_sink(
     }))
 }
 
-fn update_avro_format_with_comments(
```

Comment:
cc @mjibson You were right in hindsight. I had to move this logic to purification because apparently we re-plan upon restart and the create sink sql was getting updated with new comments when it should have been frozen.